tools/testing-server/router.js

```diff
@@ -195,7 +195,7 @@ class RouterHandle {
 
   expectIns(appID) {
     // insights harvest at 30s
-    return this.expectBeaconRequest(this.beaconRequests.ins, 40000, appID)
+    return this.expectBeaconRequest(this.beaconRequests.ins, undefined, appID)
```

Comment:
Can you just pass in the timeout as an argument to the promise, instead of mutating the default timeout for the whole test, and changing the default behavior for all ins promises across the test suite?

```javascript
expectIns(appID, timeout) {
    // insights harvest at 30s
    return this.expectBeaconRequest(this.beaconRequests.ins, timeout || 40000, appID)
}
```

Reply:
Do note that based on my test, this is actually upper bounded by the `-t 32000` we set most tests to anyways. It's just that for some odd reason it's coded in such a way that if it's a hardcoded value rather than `undefined`, it won't respect the dynamic router timeout change.